File: Frontend/src/app/modelos/local.ts
import { Distrito } from "./distrito";

export class Local {
    id_local: number;
    distrito: Distrito;
    dir_local: string;
    tel_local: string;
    email_local: string;
    horario_apertura: string;
    horario_cierre: string; 
    enable: string;

    constructor() {
        this.id_local = 0;
        this.distrito = new Distrito();
        this.dir_local = "";
        this.tel_local = "";
        this.email_local = "";
        this.horario_apertura = "";
        this.horario_cierre = "";
        this.enable = "S";
    }
}

File: Backend/The-Coffe-Palace-Feign-Clien/src/main/java/com/feign_client/data/model/Local.java
package com.feign_client.data.model;

import lombok.Data;

@Data
public class Local {
	private Long id_local;
	private Distrito distrito;
	private String dir_local;
    private String tel_local;
    private String email_local;
    private java.time.LocalTime horario_apertura;
    private java.time.LocalTime horario_cierre;
    private String enable = "S";

}
